src/main/java/com/griddynamics/finalTask/User.java

```diff
@@ -0,0 +1,118 @@
+package com.griddynamics.finalTask;
+
+import com.sun.istack.internal.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+public class User {
+    public String userId;
+    private final String firstName;
+    private final String lastName;
+    public String nickName;
```

Comment:
@kolegova-mary do not see any reasons why fields should have public access modifier, could you please explain?